### PR TITLE
optimized css, sidebar buttons, relocated signout button, and added …

### PR DESCRIPTION
**Pull request by andasan**
…the post form on timeline

### CODE REVIEW

File: src/components/Sidebar.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import styled from "styled-components";
 import { Link } from "react-router-dom";
 
+
```
Comment (taishikato) [inline src/components/Sidebar.jsx:4]: I don't think we need this new line

File: src/components/Sidebar.jsx [hunk 2]
```diff
@@ -12,33 +13,62 @@ const menus = [
...
+        >
+          <Span>{menu.name}</Span>
+        </Link>
+      ) : (menu.handler) ?(
+        <Link 
+          to={menu.path} 
+          onClick={(e) => handleClick(e)} 
+          style={{ textDecoration: "none", color: "#fff" }}
+        >
+            <Span>{menu.name}</Span>
```
Comment (taishikato) [inline src/components/Sidebar.jsx]: indent must be 2-space 👍

File: src/components/Sidebar.jsx [hunk 3]
```diff
@@ -12,33 +13,62 @@ const menus = [
...
+      {menu.path ? (
+        <Link 
+          to={menu.path} 
+          style={{ textDecoration: "none", color: "#fff" }}
+        >
+          <Span>{menu.name}</Span>
+        </Link>
+      ) : (menu.handler) ?(
+        <Link 
+          to={menu.path}
```
Comment (taishikato) [inline src/components/Sidebar.jsx]: There is a `path` item when menu has a `handler`?
```javascript
const menus = [
  {
    name: "",
    icon: "fab fa-twitter"
  },
  {
    name: "Home",
    icon: "fas fa-home",
    path: "/timeline"
  },
  {
    name: "Profile",
    icon: "far fa-user",
    path: "/profile"
  },
  {
    name: "Logout",
    icon: "fas fa-sign-out-alt",
    handler: "out"
  }
];
```

Comment (taishikato) [reply src/components/Sidebar.jsx]: I reckon that you can just use a button element and redirect users to other URLs inside the `handleClick` function.

Comment (taishikato) [reply src/components/Sidebar.jsx]: I mean, we should use the button if that does not need a `href` attr

Comment (andasan) [reply src/components/Sidebar.jsx]: just followed the flow of how it was made.
removed Logout from the menus object and created an independent link after the map.

File: src/components/Sidebar.jsx (hunk 2 above)
Comment (andasan) [reply src/components/Sidebar.jsx]: rectified

File: src/components/Sidebar.jsx (hunk 1 above)
Comment (andasan) [reply src/components/Sidebar.jsx:4]: rectified

**Comment by taishikato** [comment]
LGTM

**Review by taishikato — APPROVED** [review]
LGTM

**Pull request merged**